07_procfs/orangepi_board_loading.txt

```diff
@@ -0,0 +1,328 @@
+[    1.536700] hub 3-0:1.0: 1 port detected
+[    1.541634] ohci-platform 1c1b400.usb: Generic Platform OHCI controller
+[    1.548286] ohci-platform 1c1b400.usb: new USB bus registered, assigned bus number 4
+[    1.556254] ohci-platform 1c1b400.usb: irq 39, io mem 0x01c1b400
+[    1.630715] hub 4-0:1.0: USB hub found
+[    1.634495] hub 4-0:1.0: 1 port detected
+[    1.639384] usb_phy_generic usb_phy_generic.1.auto: supply vcc not found, using dummy regulator
+[    1.652626] cfg80211: Loading compiled-in X.509 certificates for regulatory database
+[    1.660734] sunxi-mmc 1c10000.mmc: allocated mmc-pwrseq
+[    1.668155] cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
+[    1.674832] ALSA device list:
+[    1.677829]   No soundcards found.
+[    1.681791] platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
+[regulatory.db
+[    1.908680] sunxi-mmc 1c10000.mmc: initialized, max. request size: 16384 KB
+[    1.922525] EXT4-fs (mmcblk0p2): INFO: recovery required on readonly filesystem
+[    1.929869] EXT4-fs (mmcblk0p2): write access will be enabled during recovery
+[    1.931124] mmc1: new high speed SDIO card at address 0001
+[    1.973707] random: fast init done
+[    2.011629] EXT4-fs (mmcblk0p2): recovery complete
+[    2.019102] EXT4-fs (mmcblk0p2): mounted filesystem with ordered data mode. Opts: (null)
+[    2.027302] VFS: Mounted root (ext4 filesystem) readonly on device 179:2.
+[    2.043412] devtmpfs: mounted
+[    2.048103] Freeing unused kernel memory: 1024K
+[    2.057831] Run /sbin/init as init process
+[    2.144862] EXT4-fs (mmcblk0p2): re-mounted. Opts: (null)
+Starting syslogd: OK
+Starting klogd: OK
+Running sysctl: OK
+Starting mdev... OK
+[    3.079752] xradio_wlan: loading out-of-tree module taints kernel.
+[    3.317856] xradio_wlan mmc1:0001:1:    Input buffers: 30 x 1632 bytes
+[    3.317856]    Hardware: 7.9
+[    3.317856]    WSM firmware ver: 8, build: 43, api: 1060, cap: 0x0003
+[    3.333710] xradio_wlan mmc1:0001:1: Firmware Label:XR_C01.08.0043 Jun  6 2016 20:41:04
+Initializing random number generator: OK
+Saving random seed: [    3.385919] random: dd: uninitialized urandom read (512 bytes read)
+OK
+Starting network: [    3.482499] dwmac-sun8i 1c30000.ethernet eth0: PHY [0.1:01] driver [Generic PHY] (irq=POLL)
+[    3.492345] dwmac-sun8i 1c30000.ethernet eth0: No Safety Features support found
+[    3.499716] dwmac-sun8i 1c30000.ethernet eth0: No MAC Management Counters available
+[    3.507411] dwmac-sun8i 1c30000.ethernet eth0: PTP not supported by HW
+[    3.527800] dwmac-sun8i 1c30000.ethernet eth0: configuring for phy/mii link mode
+udhcpc: started, v1.33.1
+[    3.578812] random: mktemp: uninitialized urandom read (6 bytes read)
+udhcpc: sending discover
+udhcpc: sending discover
+udhcpc: sending discover
+udhcpc: no lease, failing
+FAIL
+Starting dropbear sshd: OK
+
+Welcome to Buildroot for the Orange Pi Zero
+OrangePi_Zero login: root
+# 
```

Comment:
This is just a dry run of the board and doesn't contain any info about module running

Reply:
Added separate file  with the module launching logs